packages/@aws-cdk/region-settings/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@aws-cdk/region-settings",
```

Comment:
thinking about this name... i am wondering if we want this to be specific to "region"... are really all those settings centered around regions? if indeed, I don't think the word "settings" really adds any information here. I'd just go with `@aws-cdk/regions`. Alternative options: `@aws-cdk/awsconfig` or `@aws-cdk/awstopology` or `@aws-cdk/awsstatic`... not sure... Maybe `@aws-cdk/regions` is the best name...